packages/linuxBrew/yugabyte.rb

```diff
@@ -0,0 +1,21 @@
+class Yugabyte < Formula
```

Comment:
I think this should be contributed directly to https://github.com/Homebrew/linuxbrew-core